docs/core/diagnostics/distributed-tracing-collection-walkthroughs.md

```diff
@@ -0,0 +1,326 @@
+                .AddConsoleExporter()
+                .Build();
+
+            await DoSomeWork();
+            Console.WriteLine("Example work done");
+        }
+```
+
+Now the app logs distributed trace information to the console:
+```dotnetcli
+> dotnet run
+Activity.Id:          00-35c0e68b0dac3c49be08a9d9cab32579-0b7477e11aa20d40-01
+Activity.ParentId:    00-35c0e68b0dac3c49be08a9d9cab32579-d95f666d24193f40-01
+Activity.DisplayName: StepOne
+Activity.Kind:        Internal
+Activity.StartTime:   2021-03-15T01:58:10.7661575Z
+Activity.Duration:    00:00:00.5013965
+Resource associated with Activity:
+    service.name: unknown_service:temp
+
+Activity.Id:          00-35c0e68b0dac3c49be08a9d9cab32579-7ba0fc6d480c9841-01
+Activity.ParentId:    00-35c0e68b0dac3c49be08a9d9cab32579-d95f666d24193f40-01
+Activity.DisplayName: StepTwo
+Activity.Kind:        Internal
+Activity.StartTime:   2021-03-15T01:58:11.2894853Z
+Activity.Duration:    00:00:01.0188689
+Resource associated with Activity:
+    service.name: unknown_service:temp
+
+Activity.Id:          00-35c0e68b0dac3c49be08a9d9cab32579-d95f666d24193f40-01
+Activity.DisplayName: SomeWork
+Activity.Kind:        Internal
+Activity.StartTime:   2021-03-15T01:58:10.7647839Z
+Activity.Duration:    00:00:01.5450288
+Resource associated with Activity:
+    service.name: unknown_service:temp
+
+Example work done
+```
+
+The console exporter is helpful for quick examples or local development but in a production deployment
+you will probably want to send logs to a central logging store. OpenTelemetry supports a variety
+of different logging destinations using different
+[exporters](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#exporter-library).
+See the [OpenTelemetry getting started](https://github.com/open-telemetry/opentelemetry-dotnet#getting-started)
+instructions for more information on configuring OpenTelemetry.
+
+## Collect using Application Insights
+
+Distributed tracing telemetry is automatically captured after configuring the Application Insights SDK 
+([.NET](https://docs.microsoft.com/en-us/azure/azure-monitor/app/asp-net), [.NET Core](https://docs.microsoft.com/en-us/azure/azure-monitor/app/asp-net-core))
+or by enabling [code-less instrumentation](https://docs.microsoft.com/en-us/azure/azure-monitor/app/codeless-overview).
+
+See the [Application Insights distributed tracing documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/app/distributed-tracing) for more
+information.
+
+> [!NOTE]
+> Currently Application Insights only supports collecting specific well-known Activity instrumentation and will ignore new user added Activities. Application
+> Insights offers [TrackDependency](https://docs.microsoft.com/en-us/azure/azure-monitor/app/api-custom-events-metrics#trackdependency) as a vendor
+> specific API for adding custom distributed tracing information.
+
+
+## Collect using a custom logging implementation
```

Comment:
Could we not use the word logging here? I was confused for a second on why we're talking about logging here 😄

Reply:
Reworded : )